### PR TITLE
Upgrade to capistrano 3 and simplify

### PR DESCRIPTION
**Pull request by mbklein**
Capistrano 3.x does a lot of things for us that we were having to do manually (and clunkily) under Capistrano 2.x.

### CODE REVIEW

File: config/browse_everything_providers.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#
```
Comment (cjcolvar) [inline config/browse_everything_providers.yml]: Did you mean to include this in the PR?

**Comment by cjcolvar** [comment]
With this deployed on spruce, I can't get to the rails console anymore.
@carrickr found this which might be helpful: https://stackoverflow.com/questions/39752623/accessing-rails-console-after-deploy-to-server

**Comment by mbklein** [comment]
The problem is that the avalon user is defaulting to the system ruby, 1.8.7. You need to run `rvm default` before trying to load the console. I don't know why it's doing that; I switched to rbenv on my dev machine months ago.

File: config/browse_everything_providers.yml (hunk 1 above)
Comment (mbklein) [inline config/browse_everything_providers.yml]: Nope. I'll remove it.

**Comment by cjcolvar** [comment]
You're right, I see that I'm not currently on any of the rvm-listed rubies.

**Comment by phuongdh** [comment]
I've put in the workaround to always run `rvm default` on login for avalon user.

**Pull request merged**